my-code/src/store/modules/movie/reducer.js

```diff
@@ -0,0 +1,18 @@
+export default function movieReducer(
+  state = { totalResults: 0, page: 0, pageCount: 0, searchStr: '', list: [] },
```

Comment:
You might create a constant to this default value and use it again on [here](https://github.com/Significa/frontend-challenge/pull/12/files#diff-0895afad1f2e6842d750e76cb2655ecdR74) and [here](https://github.com/Significa/frontend-challenge/pull/12/files#diff-efa9ccd18515e4f1a53fb7bd651d61e3R23)